controllers/admin.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
+const Product = require('../models/product');
+
+exports.getAddProduct = (req, res) => {
+    res.render('admin/edit-product', {
+      pageTitle: 'Add Product',
+      path: '/admin/add-product',
+      editing: false
+    });
+}
+
+exports.postAddProduct = (req, res) => {
```

Comment:
instead of exporting this function directly, make it constant first, and at the end of the file, export all the functions together.